docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 
-   This action copies the default or the shortened URL to the clipboard.
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. In the **Link access** drop-down, select **Anyone with the link**.
+1. Click the checkbox confirming that you understand payment is required to add users.
+1. Click **Accept**.
+1. In the **Invite** field, enter the email address of the person you want to invite and click **Invite**.
 
-1. Send the copied URL to a Grafana user with authorization to view the link.
+   You can only invite one person at a time.
 
-### Publish a snapshot
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. Click **Copy external link**.
+1. Send the copied URL to any external user.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-A dashboard snapshot shares an interactive dashboard publicly. Grafana strips sensitive data such as queries (metric, template and annotation) and panel links, leaving only the visible metric data and series names embedded in the dashboard. Dashboard snapshots can be accessed by anyone with the link.
+### Share externally to anyone with a link
 
-You can publish snapshots to your local instance or to [snapshots.raintank.io](http://snapshots.raintank.io). The latter is a free service provided by Grafana Labs that enables you to publish dashboard snapshots to an external Grafana instance. Anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
+To share your dashboard so that anyone with the link can access it, follow these steps.
 
-1. Click the **Snapshot** tab.
-1. Click **Publish to snapshots.raintank.io** or **Publish Snapshot**.
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. In the **Link access** drop-down, select **Anyone with the link**.
+1. Click the checkbox confirming that you understand the entire dashboard will be public.
+1. Click **Accept**.
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. Each recipient will receive an email with link to the dashboard that's valid for 30 days.
```

Comment:
this part is not ok. there's no email or valid dashboard URL for 30 days. Anyone with the URL (that has to be copy pasted) will have access to the dashboard, until the user decides to "revoke access" (delete the public dashboard)